facebook-commerce-events-tracker.php

```diff
@@ -1258,6 +1267,32 @@ public function get_tracked_events() {
 			return $this->tracked_events;
 		}
 
+		/**
+		 * Gets the pending events awaiting to be sent
+		 *
+		 * @return array
+		 */
+		public function get_pending_events() {
+			return $this->pending_events;
+		}
+
+		/**
+		 * Send pending events.
+		 */
+		public function send_pending_events() {
+
+			$pending_events = facebook_for_woocommerce()->get_integration()->events_tracker->get_pending_events();
```

Comment:
Why not just use `$this->get_pending_events()`, we are in the same class and object?
```suggestion
			$pending_events = $this->get_pending_events();
```